modules/nf-core/fgbio/callduplexconsensusreads/main.nf

```diff
@@ -4,26 +4,24 @@ process FGBIO_CALLDUPLEXCONSENSUSREADS {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/fgbio:2.0.2--hdfd78af_0' :
-        'biocontainers/fgbio:2.0.2--hdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/fgbio:2.2.1--hdfd78af_0' :
+        'biocontainers/fgbio:2.2.1--hdfd78af_0' }"
 
     input:
     tuple val(meta), path(bam)
-    // please note:
-    // --min-reads is a required argument with no default
-    // --min-input-base-quality is a required argument with no default
-    // make sure they are specified via ext.args in your config
+    val min_reads
+    val min_baseq
 
     output:
     tuple val(meta), path("${prefix}.bam"), emit: bam
-    path "versions.yml"                     , emit: versions
+    path "versions.yml"                   , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
-    prefix = task.ext.prefix ?: "${meta.id}_consensus"
+    prefix = task.ext.prefix ?: "${meta.id}_consensus_unmapped"
```

Comment:
```suggestion
    prefix = task.ext.prefix ?: "${meta.id}"
```

Reply:
I am curious why remove the suffix?  This can cause name collisions when naively publishing, as well as I think the suffix makes the file a lot more descriptive.

Reply:
The guidelines state that the default `prefix` should be the `meta.id`. It's up to the pipeline developer to make the prefix more descriptive since this can differ from pipeline to pipeline.

Reply:
Gotcha. . I’d argue this is a bit of a foot gun, but if the _“Guidelines”_ would stop this PR from being merged, I’d understand.

Reply:
I would definitely agree that it is a foot gun, and I'm working on altering the guidelines ;)

Reply:
this usually goes in the config, and it should be set this way on the nf-test config. 
@nh13 highlights a quite painful point, which we probably haven't succeeded documenting well for the casual user, i.e. that test configs represent an example for implementation :)